community/i3wm/APKBUILD

```diff
@@ -1,39 +1,39 @@
 # Contributor: Sören Tempel <soeren+alpine@soeren-tempel.net>
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=i3wm
-pkgver=4.12
-pkgrel=1
+pkgver=4.13
+pkgrel=0
 pkgdesc="A tiling window manager"
 url="http://i3wm.org"
 arch="all"
 license="BSD"
-depends=""
-depends_dev=""
 makedepends="$depends_dev bison flex libxcb-dev xcb-util-cursor-dev
 	xcb-util-keysyms-dev xcb-util-wm-dev libev-dev pango-dev cairo-dev
-	yajl-dev startup-notification-dev pcre-dev libxkbcommon-dev"
-install=""
+	yajl-dev startup-notification-dev pcre-dev libxkbcommon-dev xcb-util-xrm-dev"
 subpackages="$pkgname-doc"
 source="http://i3wm.org/downloads/i3-${pkgver}.tar.bz2
 	musl.patch"
 
-builddir="$srcdir/i3-$pkgver"
+builddir="$srcdir/i3-$pkgver/"
+
 build() {
 	cd "$builddir"
-	make || return 1
+    	./configure --disable-builddir --prefix="/usr"
+	make
 }
 
 package() {
 	cd "$builddir"
-	make DESTDIR="$pkgdir/" install || return 1
+	make DESTDIR="$pkgdir/" install 
 
 	install -d "$pkgdir/usr/share/man/man1"
-	install -m644 man/*.1 "$pkgdir"/usr/share/man/man1/ || return 1
+	install -m644 man/*.1 "$pkgdir"/usr/share/man/man1/ 
+}
+
+check() {
+	cd "$builddir"
+	i3 --version > /dev/null
```

Comment:
This should refer to the compiled i3 in `$builddir`, as `check()` runs right after `build()`. Then the tests should work. Also, no need to redirect to `/dev/null` for this.